Source/WebKit/UIProcess/Automation/WebAutomationSession.h

```diff
@@ -172,6 +172,7 @@ class WebAutomationSession final : public API::ObjectImpl<API::Object::Type::Aut
     void getBrowsingContext(const Inspector::Protocol::Automation::BrowsingContextHandle&, Ref<GetBrowsingContextCallback>&&);
     void createBrowsingContext(std::optional<Inspector::Protocol::Automation::BrowsingContextPresentation>&&, Ref<CreateBrowsingContextCallback>&&);
     Inspector::Protocol::ErrorStringOr<void> closeBrowsingContext(const Inspector::Protocol::Automation::BrowsingContextHandle&);
+    Inspector::Protocol::ErrorStringOr<void> deleteSession();
```

Comment:
If this never fails, I guess it could be void instead?

Reply:
Tried making it void, but my understanding is that `ErrorStringOr<...>` is a requirement of the inspector protocol for sync messages (The other methods returning void are marked as async in `Automation.json`)